index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Кэт Энерджи</title>
+</head>
+<body>
+  <header class="page-header">
+		<div class="page-header__wrapper">
+			<a class="page-header__logo logo">
+				<img class="logo__image" src="images/logo-mobile.png" width="33" height="37" alt="Логотип Cat Energy">
+        <img class="logo__modile" src="images/logo-text-mobile.svg" width="101" height="18" alt="логотип мобильный">
+			</a>
+			<nav class="page-header__main-nav main-nav main-nav--close">
+				<button class="main-nav__toggle" type="button">
+					<span class="visually-hidden">Открыть/закрыть меню</span>
+				</button>
+				<ul class="main-nav-list">
```

Comment:
Один из улучшений методологии - это избавление от этих цепочек с дефисами. Есть блок `main-nav`,  значит внутри него можем задать элементы
```suggestion
				<ul class="main-nav__list">
```
Так же с item и link